examples/platforms/efr32mg13/efr32mg13.ld

```diff
@@ -0,0 +1,229 @@
+/***************************************************************************//**
```

Comment:
A similar linker script was not needed in `examples/platforms/efr32mg12` and `examples/platforms/efr32mg21`. Is something different with the efr32mg13 that requires this linker script to be here?

If so, we'll either need to put the OpenThread license header on top or move this into `third_party/silabs`.

Reply:
mg13 linker script file deleted (modified linker scripts will be required for all efr32 platforms when we merge in the nvm3 pr at a later date, but required for the mg13 pr)